src/Matrix.cc

```diff
@@ -2492,6 +2493,18 @@ NAN_METHOD(Matrix::MeanWithMask) {
   info.GetReturnValue().Set(arr);
 }
 
+NAN_METHOD(Matrix::Mean) {
+  SETUP_FUNCTION(Matrix)
+
+  cv::Scalar means = cv::mean(self->mat);
+  v8::Local<v8::Array> arr = Nan::New<Array>(3);
```

Comment:
It would be nice to not assume a 3 channel matrix here, can we determine the number of channels in the src matrix and return that many means?